backend/src/main/java/com/hackathon/backend/model/Product.java

```diff
@@ -28,7 +28,7 @@ public class Product {
     @Column(nullable = false)
     private String name;
 
-    @Enumerated(EnumType.ORDINAL)
+    @Enumerated(EnumType.STRING)
```

Comment:
Because we discussed this point in discord. We used to think that the number of categories was predetermined, but now we know that's not true. Therefore, we decided to create our own categories, which does not guarantee that their number will not change in the future.

Reply:
So in our case String is preferred. Maybe in the future we will create separate table for this field.